### ISSUE TITLE
Regularisation of parameters in jax / numpy

### ISSUE THREAD

**Issue by VolodyaCO** [netket/netket#505]
I have realised that, in many exercises I have done with netket, the neural quantum state correctly learns probability amplitudes up to a normalising constant. However, sometimes (almost always) the probability amplitudes are huge. I didn't have any issue until I started using convolutional neural networks in jax for a very specific problem. The issue is that netket converges to the expected ground state energy (also, it learns the correct ground state). But after some more training, the weights start to become very large, and eventually the weights go to nan.

Is there already regularisation of parameters? If not, are there plans to include this option?

**Comment by gcarleo** [comment]
I think that SGD or even some of the other optimizers already have a L2reg
parameter or similar to set up a L2 regularization . If not this is
straightforward to implement

**Comment by VolodyaCO** [comment]
Oh you are right, sorry. I didn't notice those params. Thanks.
